### PR TITLE
Extend RabbitMQ scaler to support HTTM management API protocol

### PR DESCRIPTION
**Pull request by holyketzer**
Docs for https://github.com/kedacore/keda/pull/700

### CODE REVIEW

File: content/scalers/rabbitmq-queue.md [hunk 1]
```diff
@@ -19,19 +19,28 @@ This specification describes the `rabbitmq` trigger for RabbitMQ Queue.
...
+      host: RabbitMqHost # AMQP protocol endpoint
       queueLength: '20' # Optional. Queue length target for HPA. Default: 20 messages
       queueName: testqueue
+      countUnacked: 'true' # Optional, use unacked + ready messages count
+      apiHost: RabbitApiHost # Optional HTTP managemet API endpoint
 ```
 
 The `host` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.  The resolved host should follow a format like `amqp://guest:password@localhost:5672/`
 
+`apiHost` the same, like `https://guest:password@localhost:443/vhostname`. Note it has optional vhost name after the host slash which will be used to scope API request.
```
[the remark this answers is not in the document]
Comment (holyketzer) [reply content/scalers/rabbitmq-queue.md]: `apiHost` has the similar format but for HTTP API endpoint, like `https://guest:password@localhost:443/vhostname`

[the remark this answers is not in the document]
Comment (holyketzer) [reply content/scalers/rabbitmq-queue.md]: Fixed

**Comment by holyketzer** [comment]
https://github.com/kedacore/keda/pull/700 was merged I rebased and fixed this PR

**Comment by zroubalik** [comment]
@holyketzer could you please rebase and resolve conflicts on this? So it could be merged?

**Pull request merged**